scipy/sparse/linalg/tests/test_matfuncs.py

```diff
@@ -67,6 +66,20 @@ def test_onenorm_matrix_power_nnm():
             expected = np.linalg.norm(Mp, 1)
             assert_allclose(observed, expected)
 
+def test_matrix_power():
```

Comment:
Would it help parametrizing this test on `power`

Reply:
I don't think so. The test in `test_base.py` catches the input validation, while this was intended as a correctness check across a few integers. I don't see a specific use here for parameterising that across the input powers?